packages/edit-navigation/src/store/utils.js

```diff
@@ -0,0 +1,20 @@
+export const KIND = 'root';
+export const POST_TYPE = 'postType';
+export const buildNavigationPostId = ( menuId ) =>
+	`navigation-post-${ menuId }`;
+
+export function uuidv4() {
```

Comment:
Should we use the `uuid` package instead? We already are using it in some packages e.g. `@wordpress/blocks`.